setup.py

```diff
@@ -317,13 +319,12 @@ def _check_and_fix_sdl2_mixer(f_path):
 # -----------------------------------------------------------------------------
 # extract version (simulate doc generation, kivy will be not imported)
 environ['KIVY_DOC_INCLUDE'] = '1'
-import kivy
```

Comment:
this will most likely break the build somehow. probably imports kivy prematurely or without the environment variable set.